hybrasyl/World.cs

```diff
@@ -45,12 +42,22 @@
 using System.Timers;
 using System.Xml;
 using System.Xml.Schema;
-using Hybrasyl.Scripting;
-using Castable = Hybrasyl.Castables.Castable;
-using Creature = Hybrasyl.Objects.Creature;
-using Hybrasyl.Statuses;
-using Hybrasyl.Messaging;
-using Hybrasyl.Loot;
+using Hybrasyl.Xml.Common;
+using Hybrasyl.Xml.String;
+using Hybrasyl.Xml.Nation;
+using Hybrasyl.Xml.Item;
+using Hybrasyl.Xml.ServerConfig;
+using Hybrasyl.Xml.Loot;
+
+
+// Conflicts between duplicate class names in existing object hierarchy and
+// xml are handled here
+
+using XmlMap = Hybrasyl.Xml.Map.Map;

```

Comment:
i really dislike all these Xml[Obj] namings. The goal was to eventually remove the underlying server classes and replace them with everything directly from extended xml classes.

Reply:
I think that is still the goal but there has to be a way to refer to conflicting types while we do that (slowly).